.github/workflows/android-release-metadata.yaml

```diff
@@ -14,10 +14,6 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v4
 
-      - name: Parallel submodules checkout
-        shell: bash
-        run: git submodule update --depth 1 --init --recursive --jobs=$(($(nproc) * 20))
-
```

Comment:
Is it needed? It looks that Gradle produce any tasks that need submodules

Reply:
https://github.com/organicmaps/organicmaps/actions/runs/15901865574/job/44846549141